src/main/java/Pebble/Parser.java

```diff
@@ -0,0 +1,99 @@
+package pebble;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ *  Parser class converts String inputs into usable commands and arguments
+ */
+public class Parser {
+
+    /**
+     * Parse strings into commands.
+     *
+     * @param input String inputs.
+     * @return Command output.
+     */
+    public static Command parseCommand(String input) {
+        String[] words = input.split(" ", 2);
+        String command = words[0];
+        String arguments = words.length > 1 ? words[1] : "";
+
+        switch (command) {
+        case "bye":
+            return new Command(CommandType.BYE, arguments);
+        case "list":
+            return new Command(CommandType.LIST, arguments);
+        case "mark":
+            return new Command(CommandType.MARK, arguments);
+        case "unmark":
+            return new Command(CommandType.UNMARK, arguments);
+        case "todo":
+            return new Command(CommandType.TODO, arguments);
+        case "deadline":
+            return new Command(CommandType.DEADLINE, arguments);
+        case "event":
+            return new Command(CommandType.EVENT, arguments);
+        case "delete":
+            return new Command(CommandType.DELETE, arguments);
+        case "find":
+            return new Command(CommandType.FIND, arguments);
+        default:
+            return new Command(CommandType.UNKNOWN, arguments);
+        }
```

Comment:
This looks very neat. Nice job on using switch case (unlike me I just used if else like a goon :{  )